src/screens/HomeScreen.js

```diff
@@ -63,6 +64,25 @@ export const HomeScreen = ({ navigation }) => {
   } = sections;
   const [refreshing, setRefreshing] = useState(false);
 
+  const interactionHandler = useCallback((response) => {
+    const data = response?.notification?.request?.content?.data;
+    const queryType = data?.query_type 
+      ? getQueryType(data.query_type) 
+      : undefined;
+
+    if (data?.id && queryType) {
+      // navigate to the newsItem
+      navigation.navigate({
+        routeName: 'Detail',
+        params: {
+          query: QUERY_TYPES.NEWS_ITEM,
```

Comment:
@digorath i found this! `query: QUERY_TYPES.NEWS_ITEM` is hardcoded but `queryType` should be used here, right?
`// navigate to the newsItem` was work in progress maybe and not finished? maybe a `// TODO: ...` would be helpful in this case.

furthermore the `params` is missing keys, which can be compared with the new methods in `queryDataParser.js`.
example for news items:

```
params: {
      title: newsItem.titleDetail,
      query: QUERY_TYPES.NEWS_ITEM,
      queryVariables: { id: `${newsItem.id}` },
      rootRouteName: 'NewsItems',
      shareContent: {
        message: shareMessage(newsItem, QUERY_TYPES.NEWS_ITEM)
      },
      details: newsItem
    },
```

the `rootRouteName` can be determined based on the `queryType`.

`shareContent` and `details` will be harder, as we do not have the data for a `newsItem` (or other detail object to open)